integration/tests/fargate/fargate_test.go

```diff
@@ -103,6 +103,7 @@ var _ = Describe("(Integration) Fargate", func() {
 			"--name", profileName,
 			"--namespace", kubeTest.Namespace,
 			"--labels", "run-on=fargate",
+			"--tags", "env=integration",
```

Comment:
I did test similar case using CLI, but this integration test only runs in `master` branch. So might need to verify later.

Reply:
Today integration test is failed for 2 cases, which I think it's not related. Please let me know if you other idea

https://circleci.com/gh/weaveworks/eksctl/6984?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
@marccarre I am wrong for the above ☝️ . Fixes in https://github.com/weaveworks/eksctl/pull/1953